docs/learn/encyclopedia/network-configuration/ledger-headers.mdx

```diff
@@ -35,7 +35,7 @@ Hash of the transaction set applied to the previous ledger.
 
 ### Close time
 
-The close time is a UNIX timestamp indicating when the ledger closes. Its accuracy depends on the system clock of the validator proposing the block. Consequently, SCP may confirm a close time that lags a few seconds behind or up to 60 seconds ahead.
+The close time is a UNIX timestamp indicating when the ledger closes. Its accuracy depends on the system clock of the validator proposing the block. Consequently, SCP may confirm a close time that lags a few seconds behind or up to 60 seconds ahead. It's strictly guaranteed to be greater than the close time of an earlier ledger.
```

Comment:
I think "It is strictly monotonic" would be a little less verbose

Reply:
```suggestion
The close time is a UNIX timestamp indicating when the ledger closes. Its accuracy depends on the system clock of the validator proposing the block. Consequently, SCP may confirm a close time that lags a few seconds behind or up to 60 seconds ahead. It's strictly monotonic – guaranteed to be greater than the close time of an earlier ledger.
```